common/djangoapps/xblock_django/user_service.py

```diff
@@ -59,7 +59,7 @@ def _convert_django_user_to_xblock_user(self, django_user):
 
         if django_user is not None and django_user.is_authenticated():
             # This full_name is dependent on edx-platform's profile implementation
-            full_name = getattr(django_user.profile, 'name') if hasattr(django_user, 'profile') else None
+            full_name = django_user.profile.name if hasattr(django_user, 'profile') else None
```

Comment:
If Python ever gets the null-aware attribute access operator that was floating around python ideas, then it can be a one-liner.